data_base.py

```diff
@@ -0,0 +1,67 @@
+import sqlite3
+from functools import wraps as wraps
+import random
+
+
+def work_with_db(file):
```

Comment:
This approach is clever, but it will be inconvenient to configure it (for example, to use separate db file in tests).
A more common implementation is to use a class that initializes a connection in __init__ and then has a bunch of methods. Users of this DB class receive an instance of it outside (to allow composition approach), and call methods on it.